Fable.Remoting.ClientV2/Http.fs

```diff
@@ -75,6 +82,8 @@ module Http =
             for (key, value) in req.Headers do
                 xhr.setRequestHeader(key, value)
 
+            xhr.withCredentials <- true
```

Comment:
Shouldn't this become the following?
```fs
xhr.withCredentials <- req.WithCredentials 
```

Reply:
Oh yea, oops! I implemented it without the boolean as an empty case at first, but forgot to change this.